### DISCUSSION TITLE
Pengaturan Email

### DISCUSSION THREAD

**Discussion by sooluh** [upkoding/upkoding#16] [Ideas]
Mungkin bisa ditambahkan halaman untuk mengatur (mengaktifkan dan menonaktifkan) email berlangganan. Ini sangat membantu dan sangat bermanfaat untuk orang-orang yang mempedulikan semua isi emailnya🤪

**Comment by ekaputra07** [comment]
Terima kasih @sooluh atas masukannya.

Sebenarnya itu sudah ada dalam roadmap sekalian memperbaiki fitur notifikasi, tapi mungkin akan saya prioritaskan ini (email notifikasi) dulu.

Mungkin dalam beberapa hari kedepan sudah akan ada pengaturannya ya, nanti saya info disini kalau sudah live. 🙏

**Comment by ekaputra07** [comment]
Halo sedikit update saja, pengaturan notifikasi email sekarang sudah tersedia via settings https://www.upkoding.com/account/settings/

🙏

**Reply by sooluh** [reply]
wihh mantap bli😁
